rai-table/index.html

```diff
@@ -0,0 +1,45 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="x-ua-compatible" content="ie=edge">
+  <title>CSIS Renewing American Innovation Project | Tracking Innovation Policy</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!--DataTables.net css-->
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="https://cdn.datatables.net/v/dt/dt-1.10.20/r-2.2.3/datatables.min.css"
+    />
+    <!-- G fonts  -->
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital,wght@0,200;0,300;0,400;0,600;1,200;1,300;1,400;1,600&display=swap"
```

Comment:
```suggestion
      href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@400;700&display=swap" 
```
Heads up that I'm updating this so we're only loading in the font weights that we need (regular & bold). You always want to be careful that you're not loading more font weights than you need to. :)